### ISSUE TITLE
Custom order for decks in deckpicker

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#1391]
Originally reported on Google Code with ID 488

```

An enhancement. Possibility of arranging decks in deck picker by number of due and
new cards (or other possible criteria of arranging), so as to have a visual feedback
of what decks are more urgent dealing with.

```

Reported by `Villatonil` on 2011-03-01 09:06:35

**Comment by daikazoku** [comment]
This doesn't seem to be fixed in v2.5.4. I get the white flashing both in night mode and in day mode. My cards have background-color: black in their css.

**Comment by daikazoku** [comment]
Mmm, weird. It seems to happen on my tablet but not on my phone.
